www/templates/html/Media.tpl.php

```diff
@@ -128,7 +130,17 @@
                         
                     </div>  
                 </div>          
-
+                <hr>
+                
+                <?php if($getTracks){
+                    echo '<div class="mediahub-onpage-captions">';
+                    echo $savvy->render($context, 'MediaPlayer/Transcript.tpl.php');
+                    echo '</div>';
+                    }
+                ?>
+               
+                <hr>
+                <h6 class="wdn-sans-serif">Description</h6>
```

Comment:
Sure. For semantics and a11y, heading elements need to be in order (none skipped), and shouldn't be used for their sizes (that can be adjusted with css). The general rule is if it gets touched, it should be brought up to standard.